src/wui/netif_settings.c

```diff
@@ -0,0 +1,185 @@
+#include "netif_settings.h"
```

Comment:
all info from EEPROM have to be obtained using interface function defined in wui_api